portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase

```diff
@@ -4467,6 +4467,87 @@ definition {
 			uploadFileName = "Document_1.jpg");
 	}
 
+	@priority = "4"
+	test SubmitFormWithUploadFieldAsGuestUser {
+		property portal.acceptance = "quarantine";
+		property portal.upstream = "quarantine";
+		property test.name.skip.portal.instance = "Forms#SubmitFormWithUploadFieldAsGuestUser";
```

Comment:
@natocesarrego ,
I think it would be interesting to have this execution isolated once if there is any document already with this same name already uploaded on Documents and Media, it would be automatically renamed with a "(1)", and the test would fail.

Example: One test had previously upload the standard document that is called "Document_1.jpg". Once uploaded again, it would be renamed to "Document_1(1).jpg".